addon.json

```diff
@@ -1,68 +1,71 @@
-                  "default": "toolbar=0,location=0,left=800,top=400,height=1000,width=1400"
-                },
-                "reuseWindow": {
-                  "type": "boolean",
-                  "default": true
-                },
-                "includeWorkbookParameters": {
-                  "type": "boolean",
-                  "default": true
-                }
+  "identifier": "com.seeq.addon.correlation",
+  "name": "Correlation Analysis",
+  "description": "Determine cross correlations and time shifts to maximize correlations among signals",
+  "version": "Input version here",
+  "maintainer": "Seeq Corporation https://seeq.com/",
+  "license": "Apache-2.0 license",
+  "icon": "fa fa-th",
+  "tags": {
+    "documentation": "Documentation website here https://seeq12.github.io/seeq-correlation/introduction.html"
+  },
+  "previews": ["additional_content/LargeMatrixExample.png"],
+  "elements": [
+    {
+      "name": "Correlation Analysis",
+      "description": "Determine cross correlations and time shifts to maximize correlations among signals",
+      "identifier": "com.seeq.addon.correlation.correlation",
+      "known_aliases": ["Correlation"],
+      "type": "AddOnTool",
+      "path": "data-lab-functions",
+      "notebook_file_path": "correlation_analysis_master.ipynb",
+      "extensions": ["ipyvuetify", "widgetsnbextension", "ipyvue"],
+      "configuration_schema": {
+        "type": "object",
+        "properties": {
+          "display": {
+            "type": "object",
+            "properties": {
+              "icon": {
+                "type": "string",
+                "default": "fa fa-th"
               },
-              "required": [
-                "icon",
-                "linkType",
-                "sortKey",
-                "windowDetails",
-                "reuseWindow",
-                "includeWorkbookParameters"
-              ]
-            }
+              "linkType": {
+                "enum": ["window", "tab", "none"],
+                "default": "window"
+              },
+              "sortKey": {
+                "type": "string",
+                "default": "c"
+              },
+              "windowDetails": {
+                "type": "string",
+                "default": "toolbar=0,location=0,left=800,top=400,height=1000,width=1400"
+              },
+              "reuseWindow": {
+                "type": "boolean",
+                "default": true
+              },
+              "includeWorkbookParameters": {
+                "type": "boolean",
+                "default": true
+              }
+            },
+            "required": [
+              "icon",
+              "linkType",
+              "sortKey",
+              "windowDetails",
+              "reuseWindow",
+              "includeWorkbookParameters"
+            ]
           },
-          "required": ["display"]
-        }
+          "advanced_project_configuration": {
+            "kernel_name": "python38"
```

Comment:
Made the change, however, still defaulting to 38 since the notebooks are still on 38 and also since the [311 pr](https://github.com/seeq12/seeq-correlation/pull/51) which takes care of both situation is yet to merge.